tests/common/test_cgroupstelemetry.py

```diff
@@ -429,6 +435,9 @@ def test_telemetry_with_tracked_cgroup(self):
             with patch("azurelinuxagent.ga.exthandlers.ExtHandlerInstance.get_log_dir", lambda *_: self.log_dir) as patch_get_log_dir:
                 self.ext_handler_instance.launch_command(command)
 
+        self.assertTrue(CGroupsTelemetry.is_tracked("/sys/fs/cgroup/cpu/walinuxagent.extensions/foobar_1.0.0"))
```

Comment:
Maybe use the same path building as in `test_cgroupconfigurator` class, e.g.:
`self.assertTrue(CGroupsTelemetry.is_tracked(os.path.join(self.cgroups_file_system_root, "cpu", "walinuxagent.extensions/Microsoft.Compute.TestExtension_1.2.3")))`

Reply:
Discussed offline. Added a new TODO for systemd test as well.